app/portainer/settings/authentication/ldap/ldap-group-search-item/ldap-group-search-item.html

```diff
@@ -1,88 +1,78 @@
-<rd-widget>
-  <rd-widget-body>
```

Comment:
these large looking changes are replacing the rd-widget-body and rd-widget with a div because they are already in a widget. There are no other changes to the inner form, other than less indentation